### PR TITLE
fix: loading sync icons lack animation

### PR DESCRIPTION
**Pull request by JeffreyMa597**
What problem does this PR solve?
------
As title.

Ref: https://github.com/Magickbase/neuron-public-issues/issues/161

Check List
------
1、List
2、Detail

### Test
e2e Test
### Task
none

### CODE REVIEW

File: packages/neuron-ui/src/components/BalanceSyncingIcon/balanceSyncIcon.module.scss [hunk 1]
```diff
@@ -1,7 +1,17 @@
 .confirm {
   margin-left: 4px;
+  animation: spin 2s linear infinite;
+
+  @keyframes spin {
```
Comment (yanguoyu) [inline packages/neuron-ui/src/components/BalanceSyncingIcon/balanceSyncIcon.module.scss:5]: May be we can move `spin` to common scss?

Comment (JeffreyMa597) [reply packages/neuron-ui/src/components/BalanceSyncingIcon/balanceSyncIcon.module.scss]: updated.

File: packages/neuron-ui/src/styles/index.scss [hunk 2]
```diff
@@ -269,3 +269,16 @@ body {
     border-radius: 48px;
   }
 }
+
+.infinite-rotation {
```
Comment (WhiteMinds) [inline packages/neuron-ui/src/styles/index.scss:273]: For cases where class selectors need to be defined, I suggest using `css module` as much as possible.
Consider creating a new `styles/index.module.scss` or `styles/animation.module.scss` file.

File: packages/neuron-ui/src/components/BalanceSyncingIcon/index.tsx [hunk 3]
```diff
@@ -10,13 +10,13 @@ export interface BalanceSyncIconProps {
 
 const BalanceSyncIcon = ({ connectionStatus, syncStatus }: BalanceSyncIconProps) => {
   if (ConnectionStatus.Connecting === connectionStatus || ConnectionStatus.Offline === connectionStatus) {
-    return <Confirming className={styles.confirm} />
+    return <Confirming className={`${styles.confirm} infinite-rotation`} />
```
Comment (WhiteMinds) [inline packages/neuron-ui/src/components/BalanceSyncingIcon/index.tsx:13]: it can be modified to use the `clsx` function from the `utils` module.

**Review by WhiteMinds — APPROVED** [review]

File: packages/neuron-ui/src/components/BalanceSyncingIcon/index.tsx (hunk 3 above)
Comment (Keith-CY) [reply packages/neuron-ui/src/components/BalanceSyncingIcon/index.tsx:13]: When will the `Confirming` not rotate infinitely?

Comment (JeffreyMa597) [reply packages/neuron-ui/src/components/BalanceSyncingIcon/index.tsx]: > When will the `Confirming` not rotate infinitely?

Add a property to control the status.

**Review by yanguoyu — APPROVED** [review]

**Review by devchenyan — APPROVED** [review]

**Pull request merged**